tests/acceptance/acceptancesuites/filtersearchsuite.js

```diff
@@ -7,10 +7,11 @@ import {
   browserForwardButton,
   registerIE11NoCacheHook
 } from '../utils';
+import { MockedFilterSearchRequest } from '../fixtures/responses/filtersearch/search';
 import SearchRequestLogger from '../searchrequestlogger';
 
 fixture`Facets page`
-  .requestHooks(SearchRequestLogger.createVerticalSearchLogger())
+  .requestHooks([SearchRequestLogger.createVerticalSearchLogger(), MockedFilterSearchRequest])
```

Comment:
I think we also make vertical search requests in this suite so those might need to be mocked too?

Reply:
good point, will do